packages/mdc-ripple/README.md

```diff
@@ -45,8 +46,8 @@ In order to function correctly, MDC Ripple requires a _browser_ implementation o
 
 Because we rely on scoped, dynamic CSS variables, static pre-processors such as [postcss-custom-properties](https://github.com/postcss/postcss-custom-properties) will not work as an adequate polyfill ([...yet?](https://github.com/postcss/postcss-custom-properties/issues/32)).
 
-[Most modern browsers](http://caniuse.com/#feat=css-variables) support CSS variables, so MDC ripple will work just fine. In other cases, MDC ripple will _still work_ if you include it in your codebase. It will simply check if CSS variables are supported upon initialization and if they aren't, gracefully exit. The only exception to this rule is Safari, which does support CSS variables
-but unfortunately ripples are disabled for (see [below](#caveat-safari) for an explanation).
+[Most modern browsers](http://caniuse.com/#feat=css-variables) support CSS variables, so MDC ripple will work just fine. In other cases, MDC ripple will _still work_ if you include it in your codebase. It will simply check if CSS variables are supported upon initialization and if they aren't, gracefully exit. The only exceptions to this rule are Edge and Safari, which do support CSS variables
```

Comment:
I feel like this whole section is trying to remind users how many browsers we support "just fine". Users assume we support things just fine. They want documentation for what we do NOT support. So I would put just this.

Edge and Safari, even though they do [support CSS variables](http://caniuse.com/#feat=css-variables), do not support MDC Ripple. See [Safari Caveat](#caveat-safari) and [Safari Edge](#caveat-edge) for an explanation.